client/src/components/global/Utility/ScrollLock.js

```diff
@@ -0,0 +1,70 @@
+import React, { PureComponent, PropTypes } from 'react';
+import ReactDOM from 'react-dom';
+
+export default class ScrollLock extends PureComponent {
+
+  static displayName = "Utility.ScrollLock";
+
+  static propTypes = {
+    children: React.PropTypes.element.isRequired
+  };
+
+  constructor() {
+    super();
+
+    this.state = {
+      touchStart: null
+    };
+
+    this.edgeLock = this.edgeLock.bind(this);
+  }
+
+  edgeLock(scroller, delta) {
+    if (delta < 0 && scroller.scrollTop <= 0) {
+      event.preventDefault();
```

Comment:
Good catch. I had this problem recently on another project where this works in chrome (mysteriously the event still exists, maybe in a global space?) But in Firefox it does not.